src/theme/global.css

```diff
@@ -14,3 +14,46 @@
 mark.current {
     background-color: orange;
 }
+
+.tab-bar {
+    margin: 8px;
+    margin-top: 0;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    border-radius: 5px;
+}
+
+.action-button {
+    --border-radius: 5px;
+    font-size: large;
```

Comment:
I've asked Dustin for developer access on Zeplin so we can see font sizes. `large` winds up being 18px, and I think `16px` is better.

Do explicit padding here, rather than relying on ion-padding. I suggest `padding: 10px 10px`